utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -87,6 +88,19 @@ impl CliConfiguration for BenchmarkCmd {
 		}
 	}
 
+	fn base_path(&self) -> Result<Option<BasePath>> {
+		let inner = unwrap_cmd! {
+			self, cmd, cmd.base_path()
+		};
+
+		//If the base path was not provided, benchmark command shall use temporary path. Otherwise
+		//we may end up using shared path, which may be inappropriate for benchmarking.
```

Comment:
```suggestion
		// If the base path was not provided, benchmark command shall use temporary path. Otherwise
		// we may end up using shared path, which may be inappropriate for benchmarking.
```